source/posts/2015-04-12-post-foodhack-wrapup.md.erb

```diff
@@ -0,0 +1,205 @@
+
+### Wiener Tafel
+**The Challenge**  
+The Wiener Tafel can currently just hand over lists of donation from caterings, privat individuals just on a small scale to interested social institutions. 
+
+**Reason**  
+legal limitations, because they are dealing with already cooked food or private food donations. 
+
+**Problems**  
+inefficiency because of the small scale, main focus on food and hygene products
+Catering happens often at night and on the weekends, so difficult to transport from both sides 
+
+Do you have ideas to help improve the situation for the important work of the Wiener Tafel?
+
+### Lunzer Massgreisslerei
+
+Andrea Lunzer from Lunzer´s Massgreißlerei challenges you to solve a problem that small grocery shops like “Bioladen” face every day. Going bad milk and bread and other time-sensitive products. They are required products, because thats what people use on a daily base. Milk and bread have a very small margin, so they are a big risk for the shop owners. Many little stores solve that problem by ordering very few, to ensure to run out instead of having left overs.  But that´s a vicious cycle. If customers can not be sure to be able to buy their important groceries in the small store, they will chose the supermarket over the small store. Can you think of a solution, that can reduce the financial risk for small shops to be able to compete with large supermarkets in order to offer their customers reliable supply of products? 
+
+### Zero Waste Jam
+
+Zero Waste Jam’s mission, besides reducing food waste by producing jams from surplus fruits, is to raise awareness about food waste and to encourage people to live according the mantra: Reduse, Reuse, Recycle. We believe that a zero waste society is possible. To make it happen we need to create an empowering movement of people of all ages and ideologies. Our goal is to make people excited about Zero Waste. Children are a key target group to make the shift happen, now and in the future. Many habits and attitudes are formed in early years, so our challenge for this Foodhack Vienna is to create a fun approach to introduce the thought of reducing food waste to children. We are planning to develop methods, workshops and learning material for preschools. Crucial elements of this would be fun visuals on our website; illustrations, comics, animations etc. We are open for your ideas and creativity. How can we introduce children from 5 till 10 years to the idea of reducing food waste?
+
+<%= slideshow("foodhack") %>
+
+## Prizes
+
+Companies gave prizes for the winners that were focused on the winning teams continuing development of their ideas.
+
+### Grand Prize
+
+* 2 days wirsing coaching
+* 40 days working in the co-space coworking space
+* 40 hours working in the Impact Hub coworking space
```

Comment:
I don't think the Co Space and Impact Hub prizes are correct. Where did you get this from?